src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/NotifyChangeFunction.cs

```diff
@@ -49,25 +48,20 @@ await validationService
                         {
                             if (message.Immediate)
                             {
-                                var releaseStatus =
-                                    await CreateReleaseStatusAsync(message, ImmediateReleaseStartedState);
-                                await queueService.QueuePublishReleaseFilesMessage(
-                                    releaseVersionId: releaseStatus.ReleaseVersionId,
-                                    releaseStatusId: releaseStatus.Id);
-                                await queueService.QueuePublishReleaseContentMessage(
-                                    releaseVersionId: message.ReleaseVersionId,
-                                    releaseStatusId: message.ReleaseStatusId);
+                                var releasePublishingKey = await CreateReleaseStatus(message,
+                                    ReleasePublishingStatusStates.ImmediateReleaseStartedState);
+                                await queueService.QueuePublishReleaseFilesMessage(new[] { releasePublishingKey });
```

Comment:
I've renamed these now, thanks. I was also attempting to get rid of `QueueService` because it's just a wrapper around `PublisherClient` which also does some state changing stuff after the message is queued, but I think it's pretty handy to keep because 2 out of 3 methods are used in multiple places, so removing it would involve duplicating the code.